yarn.lock

```diff
@@ -10435,24 +10435,31 @@ semver-greatest-satisfied-range@^1.1.0:
 
 "semver@2 || 3 || 4 || 5", semver@^5.1.0, semver@^5.5.0, semver@^5.6.0:
   version "5.7.2"
-  resolved "https://registry.npmjs.org/semver/-/semver-5.7.2.tgz"
+  resolved "https://registry.npmjs.org/semver/-/semver-5.7.2.tgz#48d55db737c3287cd4835e17fa13feace1c41ef8"
   integrity sha512-cBznnQ9KjJqU67B52RMC65CMarK2600WFnbkcaiwWq3xy/5haFJlshgnpjovMVJ+Hff49d8GEn0b87C5pDQ10g==
 
 semver@7.0.0:
```

Comment:
This still appears to have semver 7.0.0 in the lockfile.

Reply:
Hey @awharn after running `npm ls semver` I found that the only semver 7.0.0 is tied to a dev dependency jest-html-reporter
<img width="1007" alt="Screenshot 2024-04-19 at 1 43 38 PM" src="https://github.com/zowe/zowe-explorer-vscode/assets/49491949/a51368f2-5de5-482f-bdf4-2b30a7a49b5a">